libs/wire-subsystems/src/Wire/EnterpriseLoginSubsystem/Interpreter.hs

```diff
@@ -0,0 +1,321 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+
+module Wire.EnterpriseLoginSubsystem.Interpreter
```

Comment:
i didn't check every details of this module against the RFC.  (should i?  probably...)